Upstream: under review https://git.reviewboard.kde.org/r/119572/

From 391afa58d62fb4c9000bd5b59649487aaedaab5a Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Sat, 2 Aug 2014 11:51:54 +0200
Subject: [PATCH 2/2] Part: use correct slots for QStatusbar

The correct names are showMessage(..) and clearMessage().
---
 src/part/part.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/part/part.cpp b/src/part/part.cpp
index e63c852..4c158c8 100644
--- a/src/part/part.cpp
+++ b/src/part/part.cpp
@@ -238,8 +238,8 @@ bool
 Part::start(const QUrl &url)
 {
     if (!m_started) {
-        connect(m_map, SIGNAL(mouseHover(QString)), statusBar(), SLOT(message(QString)));
-        connect(m_map, SIGNAL(created(const Folder*)), statusBar(), SLOT(clear()));
+        connect(m_map, SIGNAL(mouseHover(QString)), statusBar(), SLOT(showMessage(const QString&)));
+        connect(m_map, SIGNAL(created(const Folder*)), statusBar(), SLOT(clearMessage()));
         m_started = true;
     }
 
-- 
2.0.4

